src/helpers/arrow-styles.js

```diff
@@ -1,45 +1,35 @@
-export const arrowUp = ({color, size}) => {
-	return {
-		width: "0px",
-		height: "0px",
-		borderLeft: `${size}px solid transparent`,
-		borderRight: `${size}px solid transparent`,
-		borderBottom: `${size}px solid ${color}`,
-		position: "absolute",
-		top: `-${size}px`
-	};
+const defaultArrow = {
+  width: '0px',
```

Comment:
Sure, the airbnb styleguide suggests single quotes though, but maybe you use some other style guide?

https://github.com/airbnb/javascript#strings--quotes